### DISCUSSION TITLE
v2.13.3

### DISCUSSION THREAD

**Discussion by Mygod** [Mygod/VPNHotspot#299] [General]
Fixes.

<hr /><em>This discussion was created from the release <a href='https://github.com/Mygod/VPNHotspot/releases/tag/v2.13.3'>v2.13.3</a>.</em>

**Comment by lovetox2021** [comment]
the vpn hotspot did not work in my nexus6  with lineageos18.1 installed

**Comment by lovetox2021** [comment]
the vpn hotspot did not work in my nexus6 with lineageos18.1 installed
it show a messege
Valid config not found , please starter repeater first
but the Repeater swith is stucked at on place

**Reply by Mygod** [reply]
Sorry to hear that. Does it work with any previous version of the app since you are posting under this thread specifically? If not, it generally indicates that your ROM does not support Wi-Fi direct properly.

**Comment by lovetox2021** [comment]
i am not sure about it
but the vpn hotspot work with nexus factory rom
but it's android 7.11
it's so old
i did not want use it
i want use lineageos 18.1
it's android 11
it's fast and smooth
with good privacy protection
especially the microphone

Sent with [ProtonMail](https://protonmail.com/) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐
2021年12月17日 星期五 2:30 下午，Mygod ***@***.***> 来信：

**Reply by Mygod** [reply]
Report to your ROM developer instead.

**Comment by lovetox2021** [comment]
i will never buy it through site
that's my rule

Sent with [ProtonMail](https://protonmail.com/) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐
2021年12月18日 星期六 1:22 上午，Mygod ***@***.***> 来信：

**Comment by lovetox2021** [comment]
even the last girl i bought the iphone for her
i know who is she and i know his telegram and whatsapp

why you only want contact me with mail?

Sent with [ProtonMail](https://protonmail.com/) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐
2021年12月18日 星期六 9:43 下午，suddenkingkong ***@***.***> 来信：

**Comment by lovetox2021** [comment]
sorry
i sent wrong text

what i want to say is
in some lineageos 18.1 rom
mainly 64bit cpu complied
the vpn hotspot works fine
the vpn hotspot work abnormal in 32bit cpu complied lineageos rom

Sent with [ProtonMail](https://protonmail.com/) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐
2021年12月18日 星期六 9:47 下午，suddenkingkong ***@***.***> 来信：
